src/runtime/contrib/verilator/verilator_runtime.cc

```diff
@@ -76,19 +76,24 @@ VerilatorProfiler* VerilatorProfiler::ThreadLocal() {
   return &inst;
 }
 
-void VerilatorRuntime::LoadLibrary(const std::string& lib_name) {
-  lib_ = new VerilatorLibrary();
-  lib_->Load(lib_name);
+VerilatorRuntime::~VerilatorRuntime() {
+  auto dealloc = reinterpret_cast<VerilatorDeallocFunc>(lib_->GetSymbol("VerilatorDealloc"));
+  ICHECK(dealloc != nullptr);
+  dealloc(device_);
+  lib_->~VerilatorLibrary();
```

Comment:
just out of curiosity, why do we explicitly call the deallocator instead of `delete lib_`?